Pearson-Advance/openedx-themes
# PADV-509: Add CertPREP Theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-509

## Description

This PR adds the templates and assets for CertPREP theme on edx-platform.

## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-vue-theme` on a site.
- Compile the CertPREP theme.
- Verify that the CertPREP theme is applied correctly on Olive.
- Enable Course Operations.
- Test course operations features.

(For more information enter here: [How to enable themes)](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html#enable-themes)

This site is involved in Course Operation [(Course Operation features)](https://github.com/Pearson-Advance/course_operations/blob/master/pearson_course_operation/docs/course_operations_features_juniper.md).


## Screenshots

Log In:
![Log in](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/b42a0bcf-3797-45cb-9245-cc9017ded5d6)


Dashboard:
![dashboard](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/1c27c571-6fb2-4ba6-a337-33d7c952ccf1)

## Reviewers
